djes/models.py

```diff
@@ -192,7 +202,13 @@ def mapping(self):
         return self.__class__.search_objects.mapping
 
     @classmethod
+    def is_orphaned(cls):
+        return getattr(cls.search_objects.mapping.Meta, 'orphaned', False)
+
+    @classmethod
     def get_base_class(cls):
+        if cls.is_orphaned():
```

Comment:
For creating a separate, proxy content document that contains data we would otherwise not include for a typical piece of content